pom.xml

```diff
@@ -311,7 +312,7 @@
           <version>${fabric8-maven-plugin.version}</version>
         </plugin>
         <plugin>
-          <groupId>com.github.kongchen</groupId>
+          <groupId>dk.nykredit.swagger</groupId>
```

Comment:
Just out of curiosity: Is this a different plugin or has the plugin been moved ? And it a change, what is the reason ?